mw-webapp/src/component/table/Table.tsx

```diff
@@ -1,40 +1,47 @@
-import {useEffect, useState} from "react";
-import {
-  flexRender,
-  getCoreRowModel,
-  useReactTable,
-} from "@tanstack/react-table";
-import {columns} from "src/component/table/columns";
-import {getDayReports} from "src/dataAccessLogic/getDayReports";
+import {memo} from "react";
+import {flexRender, HeaderGroup, RowModel} from "@tanstack/react-table";
 import {DayReport} from "src/model/businessModel/DayReport";
 import styles from "src/component/table/Table.module.scss";
 
 /**
- * Table (need update for split component and logic code)
+ * Tables data
  */
-export const Table = () => {
-  const [data, setData] = useState<DayReport[]>([]);
-
-  const loadDayReports = async () => {
-    const dayReports = await getDayReports();
-    setData(dayReports);
-  };
+export interface TablesData<T> {
```

Comment:
Why do you need to export this interface? If we will use it in other files, maybe it would be better to put it to a separate file with interfaces?

Reply:
In any case, right now this interface is only used in this file. In that case, should it be exported at all?

Reply:
fix